doc/Administration-guide/System-installation/Single-node-installation/index.md

```diff
@@ -266,424 +453,690 @@ Add the MongoDB repository to the /etc/apt/sources.list.d/mongodb.list.
 
-Check that the PostgreSQL or MariaDB password and username is up to date in the server configuration files.
+You can find SQL database configuration property file templates in "/etc/kaa-node/conf/" folder: mariadb-dao.properties.template file for MariaDB database and postgresql-dao.properties.template file for PostgreSQL.
+
+<ul>
+<li style="list-style-type: none;">
+<ul class="nav nav-tabs">
+  <li class="active"><a data-toggle="tab" href="#maria_conf">MariaDB</a></li>
+  <li><a data-toggle="tab" href="#postgre_conf">PostgreSQL</a></li>
+</ul>
+
+<div class="tab-content">
+<div id="maria_conf" class="tab-pane fade in active" markdown="1" >
+
+Check that the MariaDB login and password is up to date in the server configuration files.
 
 ```bash
+$ cat /etc/kaa-node/conf/admin-dao.properties | grep jdbc_username
+jdbc_username=sqladmin
+
 $ cat /etc/kaa-node/conf/admin-dao.properties | grep jdbc_password
 jdbc_password=admin
-$ cat /etc/kaa-node/conf/dao.properties | grep jdbc_password
+
+$ cat /etc/kaa-node/conf/sql-dao.properties | grep jdbc_username
+jdbc_username=sqladmin
+
+$ cat /etc/kaa-node/conf/sql-dao.properties | grep jdbc_password
 jdbc_password=admin
+```
+In case of the password or username mismatch, edit the configuration file to set a new password.
 
-$ cat /etc/kaa-node/conf/admin-dao.properties | grep jdbc_username
-jdbc_username=admin
-$ cat /etc/kaa-node/conf/dao.properties | grep jdbc_username
-jdbc_username=admin
+```bash
+$ sudo nano /etc/kaa-node/conf/admin-dao.properties
+
+$ sudo nano /etc/kaa-node/conf/sql-dao.properties
+```
+
+</div><div id="postgre_conf" class="tab-pane fade" markdown="1" >
+
+Check that the PostgreSQL password is up to date in the server configuration files.
+
+```bash
+$ cat /etc/kaa-node/conf/admin-dao.properties | grep jdbc_password
+jdbc_password=admin
+
+$ cat /etc/kaa-node/conf/sql-dao.properties | grep jdbc_password
+jdbc_password=admin
 ```
 In case of the password or username mismatch, edit the configuration file to set a new password.
 
 ```bash
 $ sudo nano /etc/kaa-node/conf/admin-dao.properties
-$ sudo nano /etc/kaa-node/conf/dao.properties
+
+$ sudo nano /etc/kaa-node/conf/sql-dao.properties
 ```
 
+</div>
+</div>
+</li>
+</ul>
+
+#### Switching between MariaDB and PostgreSQL
+
+To switch between databases change contents of sql-dao.properties and admin-dao.properties files (see /etc/kaa-node/conf/).
+
+<ul>
+<li style="list-style-type: none;">
+<ul class="nav nav-tabs">
+  <li class="active"><a data-toggle="tab" href="#maria_switch">MariaDB</a></li>
+  <li><a data-toggle="tab" href="#postgre_switch">PostgreSQL</a></li>
+</ul>
+
+<div class="tab-content">
+<div id="maria_switch" class="tab-pane fade in active" markdown="1" >
+
+<ul>
+<li style="list-style-type: none;">
+<ul class="nav nav-tabs">
+  <li class="active"><a data-toggle="tab" href="#maria_sqldao">sql-dao.properties</a></li>
+  <li><a data-toggle="tab" href="#maria_admindao">admin-dao.properties</a></li>
+</ul>
```

Comment:
Do not use tabs in this case. Content of each property file should be provided sequentially.